tensorflow_datasets/image/cityscapes.py

```diff
@@ -0,0 +1,185 @@
+# pylint: disable=bad-indentation
+
+'''Cityscapes Datasets.'''
+
+import os
+import re
+
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = '''\
+@inproceedings{Cordts2016Cityscapes,
+  title={The Cityscapes Dataset for Semantic Urban Scene Understanding},
+  author={Cordts, Marius and Omran, Mohamed and Ramos, Sebastian and Rehfeld, Timo and Enzweiler, Markus and Benenson, Rodrigo and Franke, Uwe and Roth, Stefan and Schiele, Bernt},
+  booktitle={Proc. of the IEEE Conference on Computer Vision and Pattern Recognition (CVPR)},
+  year={2016}
+}
+'''
+
+_DESCRIPTION = '''\
+  Cityscapes is a dataset consisting of diverse urban street scenes and dense pixel-level
+  annotations taken across 50 different cities at varying times of the year. The label
+  annotations span across 30 (34 including ingore indices) distinct classes of objects
+  commonly encountered in driving scenes. Detailed label information may be found here:
+  https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py#L52-L99
+
+  The dataset consists of 5000 annotated image, label pairs at 1024 * 2048 resolution
+  pre-split into training (2975) validation (500) and test (1525) sets.
```

Comment:
Please add:

WARNING: this dataset requires users to setup a login and password in order to get the files.

Reply:
Where would this go? end of the first paragraph? end of description string?

Reply:
Wherever you prefer. 
You could add it at the end.

Later it will be automatically copied to datasets.md